client/src/components/App/Content/InvestigationForm/TabManagement/PersonalInfoTab/PersonalInfoTab.tsx

```diff
@@ -295,6 +294,9 @@ const PersonalInfoTab: React.FC<Props> = ({ id }: Props): JSX.Element => {
                             step: `got errors updating status: ${error}`,
                         }))
                 }
+                personalInfoValidationSchema.isValid(data).then(valid => {
```

Comment:
why moving it to the then block?

Reply:
I will move it to finally block as you suggested above

Reply:
fixed